src/command.rs

```diff
@@ -30,6 +30,8 @@ use async_recursion::async_recursion;
 use miette::{bail, ensure, miette, Context as _, IntoDiagnostic};
 use semver::{Version, VersionReq};
 use std::{env, path::Path, str::FromStr};
+#[cfg(feature = "git")]
+use tokio::process;
```

Comment:
I'd just qualify the import in line 202 and limit the amount of `#[cfg(feature = "git")]` spread across the codebase.